__tests__/api.test.js

```diff
@@ -23,3 +25,31 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET:200 return all api informations", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const infos = body.endpoints;
+        expect(infos).toEqual(endpoints);
+        for (const info in infos) {
+          expect(infos[info]).toHaveProperty("description");
+          expect(infos[info]).toHaveProperty("queries");
```

Comment:
Not every endpoint will have those three properties! "/api" will just have a description if we look at the endpoints.json!

I think line 36 should be enough